tools/openmldb_ops.py

```diff
@@ -380,5 +598,34 @@ def ScaleIn(executor : Executor, endpoints : list):
                 ScaleIn(executor, endpoints)
             else:
                 log.error("no endpoint specified")
+    elif options.cmd == "pre-upgrade" or options.cmd == "post-upgrade":
+        if options.endpoints is None:
+            log.warning("must provide --endpoints")
+        endpoints = options.endpoints.split(",")
+        if len(endpoints) != 1:
+            log.warning("must provide --endpoints with only one endpoint")
+        if options.cmd == "pre-upgrade":
+            PreUpgrade(executor, endpoints[0], options.statfile)
+        else:
+            PostUpgrade(executor, endpoints[0], options.statfile)
+    elif options.cmd == "showopstatus":
+        status, results = GetOpStatus(executor, options.db, options.filter, False)
+        if status.OK():
+            header = ["db", "op_id", "op_type", "name", "pid", "status", "start_time", "execute_time", "end_time",
+                      "cur_task", "for_replica_cluster"]
+            PrettyPrint(results, header)
+        else:
+            print(status.msg)
+    elif options.cmd == "showtablestatus":
```

Comment:
`showopstatus` and `showtablestatus` need not to disable and recover `autofailover`